File: Solutions/LeeInkyu/Game Shop/02/Armor.java
/**
 * Created by Inkyu Lee on 2017-05-28.
 */
public class Armor extends Item{
    int defense;

    public Armor(String namePm, String descriptionPm, int weightPm, int valuePm, int defensePm)
    {
        this.name = namePm;
        this.description = descriptionPm;
        this.weight = weightPm;
        this.value = valuePm;
        this.defense = defensePm;
    }

    public void describe()
    {
        super.describe();
        System.out.println("Defense     = " + this.defense);
        System.out.println();
    }
}

File: Solutions/Stareat0707/Game Shop/Game Shop, Part 2/Armor.cpp
#include "Armor.h"

Armor::Armor(std::string name, std::string description, int weight, int value, int defense) : Item(name, description, weight, value), defense(defense)
{
}

void Armor::Describe()
{
	Item::Describe();
	std::cout << "Defense       = " << defense << std::endl;
}